packages/tooling/fast-tooling/src/data-utilities/mapping.vscode-html-languageservice.ts

```diff
@@ -28,41 +28,43 @@ function mapAttributesAndSlotsToData(
     schemaId: string,
     schemaDictionary: SchemaDictionary
 ) {
-    return Object.entries(node.attributes)
-        .map(([attributeKey, attributeValue]: [string, string]) => {
-            if (schemaDictionary[schemaId].properties[attributeKey]) {
-                if (
-                    schemaDictionary[schemaId].properties[attributeKey].type ===
-                    DataType.boolean
-                ) {
-                    // When the attribute is a boolean, it does not matter
-                    // what it's value, it will always be true if present
-                    return [attributeKey, true];
-                }
+    if (schemaId !== null && schemaId !== undefined) {
```

Comment:
Probably didn't need to change this file, will undo this change, check it and push up